server/tsuserver.py

```diff
@@ -242,6 +242,7 @@ def broadcast_global(self, client, msg, as_mod=False):
                 'GLOBAL#{}#{}#{}#{}'.format(int(as_mod), client.area.id, char_name, msg))
 
     def send_modchat(self, client, msg):
+        char_name = client.get_char_name()
```

Comment:
maybe we should just take a moment to strip out all of the district code. nobody uses that anymore